### DISCUSSION TITLE
Which MirthConnect version supports log4j-1.2.17

### DISCUSSION THREAD

**Discussion by sal4455** [nextgenhealthcare/connect#5260] [General]
Which version of MirthConnect is using log4j 1.2.17 ?

**Comment by jonbartels** [comment]
None. Even the latest HEAD from `development` still has 1.2.16. You can see this in the various `lib` directories, https://github.com/nextgenhealthcare/connect/tree/development/server/lib 

The big discussion thread at https://github.com/nextgenhealthcare/connect/discussions/4892 covers the official response from NextGen.

**Comment by sal4455** [comment]
Thankyou @jonbartels
